### DISCUSSION TITLE
Personal notes for users, by which you can search

### DISCUSSION THREAD

**Discussion by Hulkmaster** [community/community#31579] [Other Feature Feedback, Questions, & Ideas]
## Problem

Some of my colleagues have pretty tricky nicknames in github and the don't wish to add their real names, so each time i have to search for them (PR/mention etc) - its a pain - i have to go to nearest issue/PR and look for their nickname

## Solution

Add ability to add personal notes to users and search by them

If user has nickname `anonymous1234567890`, and i've added note `John Doe`, then when i search for, for example, people in PR and type `John` -> `anonymous1234567890` should be also suggested

**Comment by SofianeHamlaoui** [comment]
Yes, this is a good idea to be honnest! 

I used to do this on Discord profiles so I can always remember who is who by adding the personal notes for each account.

![](https://i.imgur.com/ORp2evv.png)

**Reply by ghost** [reply]
Hi SofianeHamlaoui, great idea.

**Reply by anziaty** [reply]
Yes, it should be like a phone - there's a user number, and there's a user name that you put in your contacts
